build/root/Makefile

```diff
@@ -181,7 +181,7 @@ ifeq ($(PRINT_HELP),y)
 check test:
 	@echo "$$CHECK_TEST_HELP_INFO"
 else
-check test: generated_files
+check test: clean generated_files
```

Comment:
hi, making "clean" a dependency seems odd in a Makefile rule.
why is this needed?

Reply:
The things after test may lead next test failed or use cache, which are not what we want to see.